pagopa/pn-frontend
# Downtime logs api bff integration - pn 10738

## Short description
Integrated bff version of the downtime logs api

## List of changes proposed in this pull request
- Generated clients and models
- Removed old api client
- Updated models of the presentation layer
- Updated downtime components and pages
- Updated tests

## How to test
Login with pa, pf and pg -> go to app status page and check that everything is ok (compare with version on test env) -> download a legal fact
Login with pa, pf and pg -> go to notification detail page, section disservizi, and check that everything is ok (compare with version on test env) -> download a legal fact
